main.html

```diff
@@ -351,15 +381,77 @@ <h2>
             </ul>
 
               <button type="submit">
-                Get in Touch
+                Get in touch!
               </button>
           </form>
 
+          <form
+                  class="contact__form desktop__form"
+                  action="https://formspree.io/f/mgebdlbl"
+                  method="POST"
+          >
+            <ul>
+
+              <li class="contact__form__element name">
+                <label class="label" for="firstname">First Name:</label>
+                <input
+                        type="text"
+                        id="firstname"
+                        name="firstname"
+                        placeholder="First Name"
+                        maxlength="30"
+                        pattern="[a-zA-Z]+"
+                        required
+                />
+              </li>
+
+              <li class="contact__form__element lastname">
+                <label class="label" for="lastname">Last Name:</label>
+                <input
+                        type="text"
+                        id="lastname"
+                        name="lastname"
+                        placeholder="Last Name"
+                        maxlength="30"
+                        pattern="[a-zA-Z]+"
+                        required
+                />
+              </li>
+
+              <li class="contact__form__element email">
+                <label class="label" for="email">Email Address:</label>
+                <input
+                        type="email"
+                        id="email"
+                        name="email"
+                        placeholder="Email Address"
+                        required
+                />
+              </li>
+
+              <li class="contact__form__element message">
+                <label class="label" for="message">Your Message:</label>
+                <textarea
+                        id="message"
+                        name="message"
+                        placeholder="Hey I'd like to connect to..."
+                        maxlength="500"
+                        required
+                ></textarea>
+              </li>
+            </ul>
+
+            <button type="submit">
+              Get in touch!
+            </button>
+          </form>
```

Comment:
Please kindly work on the `contact me` section following the design given, following the guidelines for the alignment of the two contents ie the writeup and form, and ensure all background images are present ie, especially the ones to the left 😄 

**Important note:** There should be one input for Full name (just like the mobile version) and this should take the space of both First Name and Last Name inputs (shown in the desktop version).

Figma design :

![image](https://github.com/rudicarrilloypr/portfolio/assets/20537712/c4fae13b-cee6-4655-a2ee-e36a4bcdb2fd)

Your design :

https://github.com/rudicarrilloypr/portfolio/assets/20537712/696c991f-41d4-4519-9300-876d5dec0a7c

Reply:
this is how it looks on the code i pushed

<img width="1224" alt="Captura de pantalla 2023-05-11 a la(s) 15 36 57" src="https://github.com/rudicarrilloypr/portfolio/assets/125106965/a3ccdf25-4120-4994-838b-bd459c07baaa">